### ISSUE TITLE
[prefer-ts-expect-error] ignores block comments

### ISSUE THREAD

**Issue by MhMadHamster** [typescript-eslint/typescript-eslint#2531]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/prefer-ts-expect-error": "error",
  }
}
```

```TS
{/* @ts-ignore */}
const x: string = 1;
```

**Expected Result**
Error that `@ts-expect-error` should be used instead of `@ts-ignore`.

**Actual Result**
No error.

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.1.1-alpha.7` |
| `@typescript-eslint/parser`        | `4.1.0` |
| `TypeScript`                       | `4.0.2` |
| `ESLint`                           | `7.8.1` |
| `node`                             | `12.13.0` |

**Comment by bradzacher** [comment]
It looks like this functionality was added in 3.9.

Prior to 3.9, the only ignore comments allowed were line comments. [repl](https://www.typescriptlang.org/play?ts=3.8.3&t=true#code/PQKgBAAgLgzgtASwOYDsD2AnApmEwBQAxmijFGAB4BcYZGCKSYAvGAIwDc+wwksiqTFiIkyYAJ406DJq074gA)
3.9+ now allow ignore comments as block comments. [repl](https://www.typescriptlang.org/play?ts=3.9.2#code/PQKgBAAgLgzgtASwOYDsD2AnApmEwBQAxmijFGAB4BcYZGCKSYAvGAIwDc+wwksiqTFiIkyYAJ406DJq074gA)

Happy to accept a PR

**Comment by MhMadHamster** [comment]
FYI prior to 3.9 advice from TS team was to use this;
```TS
{/*
 // @ts-ignore */}
```
for tsx code. This does not work correctly with `prefer-ts-expect-error` (or `ban-ts-comment` for that matter) as well.

**Comment by yasarsid** [comment]
@bradzacher  - I could start looking at this. After initial investigation it seems easy enough - the only problem seems to be generating the `ruleFix` for block comments. 

Let raise a PR - maybe we can discuss more around there.
